docs/pages/project/contributing/contributing-docs.md

```diff
@@ -317,24 +317,6 @@ Just for fun, here are all the types:
 {% include alert.html type="secondary" content="This is a secondary alert" %}
 
 
-### Alerts
```

Comment:
There was a repetition of this "Alerts", which I have deleted

Reply:
Do me a favor here, we have added that section recently, so please add your section just after this ``Alerts`` section, with this heading `alert.html`

Reply:
okay okay